### DISCUSSION TITLE
Macos/OSX

### DISCUSSION THREAD

**Discussion by mansley12** [trizen/youtube-viewer#369] [Q&A]
Question, should I use this version or youtube.viewer for OSX/MacOS? I managed to install on Big Sur pipe-viewer CLI version and it works, whereas youtube-viewer does not. Do know of a way to correctly install the GTK dependancies?

Test Summary Report
t/00-init.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/floating-refs.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/overrides.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/signals.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/vfuncs-destroy.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/vfuncs-double-setup.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GdkEvent.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkActionGroup.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkBuilder.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkCellLayoutIface.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkCellRendererIface.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkContainer.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkDialog.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkImage.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkInfoBar.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkRadioAction.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkRadioButton.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkRadioMenuItem.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkRadioToolButton.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkRecentChooserDialog.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkTextBuffer.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
t/zz-GtkTreeModelIface.t (Wstat: 512 Tests: 0 Failed: 0)
Non-zero exit status: 2
Parse errors: No plan found in TAP output
Files=22, Tests=0, 3 wallclock secs ( 0.08 usr 0.10 sys + 2.20 cusr 0.94 csys = 3.32 CPU)
Result: FAIL
Failed 22/22 test programs. 0/0 subtests failed.
make: *** [test_dynamic] Error 2
XAOC/Gtk3-0.038.tar.gz
one dependency not OK (Glib::Object::Introspection); additionally test harness failed
/usr/bin/make test -- NOT OK
//hint// to see the cpan-testers results for installing this module, try:

**Comment by trizen** [comment]
The CLI version is known to work on MacOS ([source](https://blog.johnkrauss.com/installing-youtube-viewer/)).

However, I'm not sure if the GUI version works too. I don't have any experience with MacOS.

Maybe someone else can help.
